FabLabKasse/faucardPayment/faucard.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, parent, amount):
         assert isinstance(amount, (Decimal, float)), "PayupFAUCard: Amount to pay not Decimal or float"
```

Comment:
You are correct. Thanks for the notice. As already mentioned this should be happening in FAUcardPaymentThread aswell.